boards/arm/lora_e5_dev_board/board.cmake

```diff
@@ -5,6 +5,7 @@ board_runner_args(pyocd "--flash-opt=-O reset_type=hw")
 board_runner_args(pyocd "--flash-opt=-O connect_mode=under-reset")
 board_runner_args(jlink "--device=STM32WLE5JC" "--speed=4000" "--reset-after-load")
 board_runner_args(stm32cubeprogrammer "--port=swd" "--reset=hw")
+board_runner_args(blackmagicprobe "--gdb-serial=/dev/ttyBmpGdb" "--connect-srst")
```

Comment:
Yeah good question, it's an open source probe and the authors provide an udev rules file for having stable nodes for both the GDB endpoint and the target UART: https://github.com/blacksphere/blackmagic/blob/master/driver/99-blackmagic.rules, which I find extremely convenient since I always seem to have some other board or debugger plugged in at the same time.

I personally think that `ttyBmpGdb` should be the default, another board sets it as well (https://github.com/zephyrproject-rtos/zephyr/blob/main/boards/arm/adafruit_feather_nrf52840/board.cmake), but then the prob works just fine without the udev rules and changing the default may break it for some users that never setup the rule file and I don't really want to introduce a regression for them.

Ultimately I'm happy enough with having the flag so I can set it to what works for me on custom boards, but if you have strong opinions about the default I'm happy to change it (or drop the board override).

Reply:
Wondering if maybe an os.environ variable for overriding the default port would be a better compromise. What do you think?

Reply:
No opinion on this right now. It seems there is something to be improved in future.

Reply:
Yeah I think this needs some infrastructure discussion since something that other backend may need as well.